assignments/assignment02.rb

```diff
@@ -25,11 +35,30 @@ def to_sentence(ary)
 
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
-  # your implementation here
+  # add all the numbers together
+  sum = ary.reduce {|i, acc| i + acc }
+
+  # divide by the length, convert to floating point
+  mean = sum/ary.length.to_f
+  return mean
 end
 
 def median(ary)
-  # your implementation here
+  # find the length of the array 
+  len = ary.length
+  
+  # sort the array
+  ary.sort!
+
+  # if it's odd, return the middle number
+  if len % 2 == 1
```

Comment:
in ruby, if is an expression that returns a value. You don't need to assign the result of each branch to a local variable median, and then "return median". Instead, you can just have the if expression be the last expression in the method.